biz.aQute.launchpad/src/aQute/launchpad/BundleSpecBuilder.java

```diff
@@ -919,9 +919,11 @@ default BundleSpecBuilder addConfiguration(URL url) {
 	default Bundle install() throws Exception {
 		byte[] build = LaunchpadBuilder.workspace.build(LaunchpadBuilder.projectDir.getAbsolutePath(), x().spec);
 		String name = x().spec.bundleSymbolicName.toString();
+		String version = x().spec.bundleVersion;
+		version = (version == null) ? "0" : version;
 		ByteArrayInputStream bin = new ByteArrayInputStream(build);
 		return x().ws.getBundleContext()
-			.installBundle(name, bin);
+			.installBundle(name + ';' + version, bin);
```

Comment:
I would suggest using `-` or `_` instead of `;` to join the bsn and version;

Reply:
Ok, makes sense as that's how they normally are concatenated in urls.